package.json

```diff
@@ -12,69 +12,60 @@
     "node": ">= 6"
   },
   "scripts": {
-    "lint": "eslint index.js lib test",
-    "pretest": "npm run lint",
+    "commitlint": "commitlint --from=master",
+    "lint": "eslint --cache lib test",
+    "release": "standard-version",
+    "security": "npm audit",
     "test:only": "jest",
-    "test:coverage": "jest --coverage",
-    "test": "jest",
-    "release": "standard-version"
+    "test:watch": "jest --watch",
+    "test:coverage": "jest --collectCoverageFrom='lib/**/*.js' --coverage",
```

Comment:
I want to create `jest.config.js` like webpack-dev-server.

Reply:
I think we need do same in `webpack-default` and here